Dockerfile

```diff
@@ -36,6 +46,7 @@ ENV HADOOP_COMMON_HOME $HADOOP_HOME
 ENV HADOOP_HDFS_HOME $HADOOP_HOME
 ENV YARN_HOME $HADOOP_HOME
 ENV HADOOP_COMMON_LIB_NATIVE_DIR $HADOOP_HOME/lib/native
+ENV LD_LIBRARY_PATH $HADOOP_HOME/lib/native/:$LD_LIBRARY_PATH
```

Comment:
This is a system wide setting. it contaminates `LD_LIBRARY_PATH` for all processes in the system.
The safest path would use Hadoop only load scripts and application level scripts.
Probably, inside both `dfs.sh` and `run.sh` you should load another file with this setting.

Reply:
Actually, LD_LIBRARY_PATH is used for any Map Reduce task.
dfs.sh and run.sh doesn't make any Map Reduce task, so it is irrelevant for them.

It is possible to change LD_LIBRARY_PATH in hadoop-env.sh, but I don't see a point why to do it...